R/recipe_commit.R

```diff
@@ -37,7 +37,11 @@ orderly_commit <- function(id, name = NULL, root = NULL, locate = TRUE) {
     }
   }
   workdir <- file.path(path_draft(config$root), name, id)
-  recipe_commit(workdir, config$root)
+  
+  capture <- inherits(root, "orderly_config") && 
```

Comment:
why use `root` here and not `config`?  should just be able to write:

```
capture_log <- isTRUE(config$get_run_option("capture_log"))
```

I think.